<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>417</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Notes Notifier</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,4">
        <property name="spacing">
         <number>30</number>
        </property>
        <property name="sizeConstraint">
         <enum>QLayout::SetMinimumSize</enum>
        </property>
        <item>
         <widget class="QTextEdit" name="noteSearch">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>30</height>
           </size>
          </property>
          <property name="placeholderText">
           <string>Поиск</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="noteHeader">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>30</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="2,4">
        <property name="spacing">
         <number>30</number>
        </property>
        <item>
         <widget class="QListWidget" name="listWidget"/>
        </item>
        <item>
         <widget class="QTextEdit" name="noteText"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="2,4">
      <property name="spacing">
       <number>30</number>
      </property>
      <item>
       <widget class="QPushButton" name="addButton">
        <property name="text">
         <string>Добавить заметку</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QPushButton" name="saveButton">
            <property name="text">
             <string>Сохранить</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QCheckBox" name="checkBox">
              <property name="text">
               <string>Включить напоминание</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDateTimeEdit" name="dateTimeEdit"/>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="clearButton">
          <property name="text">
           <string>Очистить</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="deleteButton">
          <property name="text">
           <string>Удалить заметку</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>750</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Меню</string>
    </property>
    <addaction name="aboutMenu"/>
    <addaction name="referenceMenu"/>
    <addaction name="separator"/>
    <addaction name="saveToFileMenu"/>
    <addaction name="loadFromFileMenu"/>
    <addaction name="separator"/>
    <addaction name="closeMenu"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="aboutMenu">
   <property name="text">
    <string>О программе</string>
   </property>
  </action>
  <action name="referenceMenu">
   <property name="text">
    <string>Справка</string>
   </property>
  </action>
  <action name="saveToFileMenu">
   <property name="text">
    <string>Выгрузить текст в файл...</string>
   </property>
  </action>
  <action name="loadFromFileMenu">
   <property name="text">
    <string>Загрузить текст из файла...</string>
   </property>
  </action>
  <action name="closeMenu">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>closeMenu</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>341</x>
     <y>208</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
